android/capacitor-fcm/build.gradle

```diff
@@ -28,6 +28,10 @@ android {
     lintOptions {
         abortOnError false
     }
+    compileOptions {
```

Comment:
can you explain a little bit about this addition?

Reply:
By using Java 8, which is [supported on all Android devices][java8], it's possible to remove the `android` vs `androidx`-specific packages (`android.support.annotation.NonNull`), and use lambdas instead.

[java8]: https://developer.android.com/studio/write/java8-support

Reply:
interesting